firmware/controllers/can/can_dash.cpp

```diff
@@ -1293,14 +1293,16 @@ void canDashboardAim(CanCycle cycle) {
 		return;
 	}
 
-	transmitStruct<Aim5f0>(0x5f0, false);
-	transmitStruct<Aim5f1>(0x5f1, false);
-	transmitStruct<Aim5f2>(0x5f2, false);
-	transmitStruct<Aim5f3>(0x5f3, false);
-	transmitStruct<Aim5f4>(0x5f4, false);
-	transmitStruct<Aim5f5>(0x5f5, false);
-	transmitStruct<Aim5f6>(0x5f6, false);
-	transmitStruct<Aim5f7>(0x5f7, false);
+	auto canChannel = engineConfiguration->canBroadcastUseChannelTwo;
```

Comment:
Is not "canChannel" a less than ideal name for "use second channel"? also what if _soon_ more than two buses would be supported? A few of stm32f7 have three CAN buses.

Reply:
Now `canBroadcastUseChannelTwo` is a Bool, CAN 2 is a `true / 1` so when more CAN buses are added we can switch to a int which keeps the logic the same.

Reply:
So why not "size_t busIndex engineConfiguration->canBroadcastUseChannelTwo;" to keep the... technical debt most localized?

Reply:
yea, that was going to be my next comment, we should really be doing `size_t busIndex = engineConfiguration->canBroadcastUseChannelTwo ? 1 : 0`